q2galaxy/core/drivers/action.py

```diff
@@ -80,6 +80,8 @@ def _convert_arguments(signature, inputs):
                     new_list = [_convert_metadata(type_, v) for v in inputs[k]]
                 elif k in signature.inputs:
                     new_list = [sdk.Artifact.load(v) for v in inputs[k]]
+                elif v == []:
+                    new_list = signature.parameters[k].default
```

Comment:
We may want to raise an error if there is no default (i.e. the default is the signature sentinel, not a real value) as this would indicate something has gone wrong with our templating/integration with galaxy.